### PR TITLE
fix #2

### LINKED ISSUE #2: Border colors of table cells are overridden by the default style
VSCode version: 1.15.0
Extension version: 0.0.2

The border rule at [Line 530](https://github.com/mjbvz/vscode-github-markdown-preview-style/blob/12491ecd7a784a7d24aa08381234af5c56bdbe90/github-markdown.css#L530) is overridden by the default `.vscode-dark table > thead > tr > th { border-color: rgba(255, 255, 255, 0.69); }` and `.vscode-dark table > tbody > tr + tr > td { border-color: rgba(255, 255, 255, 0.18); }` rules, causing the borders are missing.

````
| Column 1 | Column 2 |
|----------|----------|
| Test     | Test     |
| Test     | Test     |
| Test     | Test     |
````

image

This a easy fix, but I don't know where does your `github-markdown.css` come from, or can I edit it directly.

### PR DESCRIPTION
**Pull request by qjebbs**
fix border colors of table

### CODE REVIEW

File: base.css [hunk 1]
```diff
@@ -30,3 +30,8 @@ body {
 .vscode-body code > div {
     background: none
 }
+
+vscode-body table th,
```
Comment (mjbvz) [inline base.css]: This needs to be a class selector: `.vscode-body`

File: base.css [hunk 2]
```diff
@@ -30,3 +30,8 @@ body {
...
+.vscode-body table td {
+    border: 1px solid #ddd !important;
```
Comment (mjbvz) [inline base.css]: Instead of adding `!important`, can you write a more specific css rule to override the existing one?

File: base.css (hunk 1 above)
Comment (qjebbs) [reply base.css]: Sorry, my mistake.

File: base.css (hunk 2 above)
Comment (qjebbs) [reply base.css]: Suppose we have markdown code:

```md
| A | B | C |
|--|--|--|
| 1 | 2 | 3 |
```

vsocde (markdown-it) renders:

```html
<article class="vscode-body">
    <table>
        <thead>
            <tr>
                <th>A</th>
                <th>B</th>
                <th>C</th>
            </tr>
        </thead>
        <tbody>
            <tr>
                <td>1</td>
                <td>2</td>
                <td>3</td>
            </tr>
        </tbody>
    </table>
</article>
```

It's so clean that we have no other attributes to assign the rules. So I cannot find another way to fix, except `!important`. If another way existed, it must be beyond my capabilities.

**Comment by mjbvz** [comment]
Thanks! I'll publish an update shortly

**Pull request merged**